qubesmanager/qube_manager.py

```diff
@@ -1318,7 +1318,17 @@ def action_open_console_triggered(self):
     def action_set_keyboard_layout_triggered(self):
         # pylint: disable=invalid-name
         for vm_info in self.get_selected_vms():
-            vm_info.vm.run('qubes-change-keyboard-layout')
+            if vm_info.vm.features.check_with_template(
+                    "supported-service.keyboard-layout"):
```

Comment:
This is now `supported-feature.keyboard-layout`. But also, add the second argument to check_with_template (default value), otherwise you'll get an exception if it is absent.